src/server/config.py

```diff
@@ -0,0 +1,3 @@
+import pathlib
+
+path_to_perception = pathlib.Path(__file__).resolve().parent.parent.parent
```

Comment:
in the smallest chance our project ever changes came, can the variable be `path_to_project_root`

Reply:
I think this isn't really something we should be accounting for at the moment. With our M.I.T. license, no one besides us should be changing the name of Perception.